src/Models/Account.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Lunanimous\Rpc\Models;
+
+class Account extends Model
+{
+    /**
+     * @var string hex-encoded 20 byte address
+     */
+    public $id;
+
+    /**
+     * @var string user friendly address (NQ-address)
+     */
+    public $address;
+
+    /**
+     * @var int balance of the account (in smallest unit)
+     */
+    public $balance;
+
+    /**
+     * @var int account type associated with the account (AccountType::Basic, AccountType::Vesting, AccountType::Htlc)
+     */
+    public $type;
```

Comment:
For Vesting Contracts and HTLCs, the result also includes the respective extra fields for those contracts:
https://github.com/nimiq/core-js/blob/03cd33f2bffb9409246427210c0223168adadd1f/clients/nodejs/modules/JsonRpcServer.js#L650-L677

These are not represented here. They should be made available, either on this model or on custom models based on the account type.

Reply:
Just to confirm, would this be a valid HTLC response ?

```json
{
    "jsonrpc": "2.0",
    "result": {
        "id": "4974636bd6d34d52b7d4a2ee4425dc2be72a2b4e",
        "address": "NQ46 NTNU QX94 MVD0 BBT0 GXAR QUHK VGNF 39ET",
        "balance": 1000000000,
        "type": 2,
        "sender": "d62d519b3478c63bdd729cf2ccb863178060c64a",
        "senderAddress": "NQ53 SQNM 36RL F333 PPBJ KKRC RE33 2X06 1HJA",
        "recipient": "f5ad55071730d3b9f05989481eefbda7324a44f8",
        "recipientAddress": "NQ41 XNNM A1QP 639T KU2R H541 VTVV LUR4 LH7Q",
        "hashRoot": "df331b3c8f8a889703092ea05503779058b7f44e71bc57176378adde424ce922",
        "hashAlgorithm": 1,
        "hashCount": 1,
        "timeout": 1105605,
        "totalAmount": 1000000000
    },
    "id": 1
}
```

Reply:
Yes, looks correct.